features/format.feature

```diff
@@ -554,3 +554,13 @@ Feature: Custom formats
         | basic_encrypted |
         | basic_folder    |
         | basic_dayone    |
+
+    Scenario: Export date counts
+        Given we use the config "datecount.yaml"
+        When we run "jrnl --export datecount"
```

Comment:
The `--export` flag has been replaced by `--format` since v2.5. The export one still works, but might be removed in the distant future. Can you change this?

Reply:
Ah! I was going off the example in #743. Let me do that.

Reply:
Fixed.